tests/Fixtures/Resumes.php

```diff
@@ -25,6 +25,15 @@ public static function createResumeWithModality($studentId, $specialization, $ta
             'modality' => $modality,
         ]);
     }
+    public static function createResumeWithoutModality($studentId, $specialization, $tagIds, $modality): Resume
```

Comment:
1) In the definition of this method, the parameter $modality is not necessary since you are not using it.

2) You should also adjust the ModalityServiceTest, where this Fixture method is being used, specifically on line that says ``$resume = Resumes::createResumeWithModality($studentId, 'frontend', ['tag1', 'tag2'], 'Presencial');``, as you are passing a value to a parameter that is not used within the method.

Reply:
done